Resources/Prototypes/_NF/Shipyard/swallow.yml

```
@@ -0,0 +1,38 @@
+# Author Info
+# GitHub: wendyribstonpippin
+# Discord: wendyribstonofpippin
+
+- type: vessel
+  id: Swallow
+  parent: BaseVessel
+  name: WS Swallow
+  description: A lightning fast microshuttle for quick and cheap transportation.
+  price: 11000
+  category: Micro
+  group: Shipyard
+  shuttlePath: /Maps/_NF/Shuttles/swallow.yml
+  guidebookPage: ShipyardSwallow
+  class:
+  - Civilian
+  engine:
+  - Plasma
+
+- type: gameMap
+  id: Swallow
+  mapName: 'WS Swallow'
+  mapPath: /Maps/_NF/Shuttles/swallow.yml
+  minPlayers: 0
+  stations:
+    Swallow:
+      stationProto: StandardFrontierVessel
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: 'Swallow {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: '14'
+        - type: StationJobs
+          availableJobs:
+            Contractor: [ 0, 0 ]
+            Pilot: [ 0, 0 ]
+            Mercenary: [ 0, 0 ]
```

```suggestion
            ContractorInterview: [ 0, 0 ]
            PilotInterview: [ 0, 0 ]
            MercenaryInterview: [ 0, 0 ]
```
Use new interview jobs.